flink-formats/flink-avro/src/test/java/org/apache/flink/formats/avro/AvroRowDataDeSerializationSchemaTest.java

```diff
@@ -205,4 +207,26 @@ public void testSpecificType() throws Exception {
 		Assert.assertEquals("12:12:12", DataFormatConverters.LocalTimeConverter.INSTANCE.toExternal(
 				rowData.getInt(2)).toString());
 	}
+
+	@Test
+	public void testIgnoreParseErrors() throws Exception {
+		DataType serDataType = ROW(FIELD("type_int", INT().notNull())).notNull();
+		RowType serRowType = (RowType) serDataType.getLogicalType();
+		AvroRowDataSerializationSchema serializationSchema = new AvroRowDataSerializationSchema(serRowType);
+		serializationSchema.open(null);
+		RowData input = GenericRowData.of(1);
+		byte[] byteData = serializationSchema.serialize(input);
```

Comment:
Would be better to using numberic type to deseriazing string type to verify the parse error. For example:
deserializing `[int: 1, string: abc]` with schema `(int, long)` should get `Row(1, null)`.

I think deserializing null field is not a strong enough test.